docs/getting-started/view-generated-code.md

```diff
@@ -0,0 +1,38 @@
+---
+id: view-generated-code
+title: Viewing generated code
+sidebar_label: View generated code
+slug: /getting-started/view-generated-code
+---
+
+# Viewing Generated Code
+
+Use  **Code View** to view and explore the generated code. You can see the updated code before it is synced with GitHub or downloaded. 
+
+
+:::note
+- Graphic files are not visible with _Code View_.
+- You cannot edit files with _Code View_.
+:::
+
+On the _Overview_ page, click **View Code**, or click the **Code View** icon on the menu.
+
+Then, select a build from the **Select build** list. 
+
+  ![](../getting-started/assets/view-code2a.png)
+
+:::note
+- The default build is the last one you created.
+- You can create a new build even if you have not made any changes.
+-  _Code View_ does not support builds created in earlier Amplication versions, before the feature had been introduced. 
+- If you perform a commit within a build, it will update the build but will not create a new build. 
+:::
```

Comment:
I didn't understand what it means

Reply:
> I didn't understand what it means

@yuval-hazaz It should have said "  If you perform a commit within a build, it will add the commit but will not create a new build. "
Is this sentence necessary, or should I remove it?

Reply:
I still dont understand it. what is "perform a commit within a build"?

Reply:
> 

It was something that @morhag90 said. I may have got it wrong.  I'll take it up with her.